bentoml/yatai/yatai_service.py

```diff
@@ -103,7 +109,12 @@ def start_yatai_service_grpc_server(
     yatai_service = YataiServicerImpl(
         db_url=db_url, repo_base_url=repo_base_url, s3_endpoint_url=s3_endpoint_url,
     )
-    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+
+    # Define interceptors here
+    grpc_interceptors = [PromServerInterceptor(), ServiceLatencyInterceptor()]
+    server = grpc.server(
+        futures.ThreadPoolExecutor(max_workers=10), interceptors=grpc_interceptors,
```

Comment:
`futures.ThreadPoolExecutor(max_workers=10)` is irrelevant here right? let's remove it. I don't think we can quite do that until https://github.com/bentoml/BentoML/issues/1540 is done.